networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,68 @@
+// External DNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= External DNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The External DNS Operator deploys and manages `ExternalDNS` to provide name resolution for services and routes from the external DNS provider to OpenShift Container Platform.
+
+These release notes track the development of the External DNS Operator in {product-title}.
+
+[id="external-dns-operator-release-notes-1.2.0"]
+== External DNS Operator 1.2.0
+
+The following advisory is available for the External DNS Operator version 1.2.0:
+
+* link:https://access.redhat.com/errata/RHEA-2023:7239[RHEA-2022:5867 ExternalDNS Operator 1.2 operator/operand containers]
+
+[id="external-dns-operator-1.2.0-new-features"]
+=== New features
+
+* The External DNS Operator now supports AWS shared VPC. For more information, see xref:../../networking/external_dns_operator/nw-creating-dns-records-on-aws.adoc#nw-control-dns-records-public-aws-with-VPC_creating-dns-records-on-aws[Creating DNS records on a public hosted zone for AWS by using Red Hat External DNS Operator].
+
+[id="external-dns-operator-1.2.0-bug-fixes"]
+=== Bug fixes
+
+* The update strategy for the operand changed from `Rolling` to `Recreate`. link:https://issues.redhat.com/browse/OCPBUGS-3630link:https://issues.redhat.com/browse/OCPBUGS-3630[OCPBUGS-3630]
+
+[id="external-dns-operator-release-notes-1.1.1"]
+== External DNS Operator 1.1.1
+
+The following advisory is available for the External DNS Operator version 1.1.1:
+
+* link:https://access.redhat.com/errata/RHEA-2024:0536[RHEA-2024:0536 ExternalDNS Operator 1.1 operator/operand containers]
+
+[id="external-dns-operator-release-notes-1.1.0"]
+== External DNS Operator 1.1.0
+
+This release included a rebase of the operand from the upstream project version 0.13.1. The following advisory is available for the External DNS Operator version 1.1.0:
+
+* link:https://access.redhat.com/errata/RHEA-2022:9086[RHEA-2022:9086-01 ExternalDNS Operator 1.1 operator/operand containers]
+
+[id="external-dns-operator-1.1.0-bug-fix"]
+=== Bug fixes
+
+* Previously, the External DNS Operator enforced empty `defaultMode` for volumes which caused conflicts with the OpenShift API defaulting. Now, the `defaultMode` is not enforced and operand deployment does not update constantly. link:https://issues.redhat.com/browse/OCPBUGS-2793[OCPBUGS-2793]
```

Comment:
Suggestion for clarity, plus another bug link formatting suggestion:

```suggestion
* Previously, the External DNS Operator enforced an empty `defaultMode` value for volumes, which caused constant updates due to a conflict with the OpenShift API. Now, the `defaultMode` value is not enforced and operand deployment does not update constantly. (link:https://issues.redhat.com/browse/OCPBUGS-2793[*OCPBUGS-2793*])
```

Reply:
yes